routes/github.ts

```diff
@@ -0,0 +1,9 @@
+import express from 'express';
+import { searchUsers, getUserProfile } from "../controller/github";
+
+const router = express.Router();
+
+router.get('/users', searchUsers);
+router.get('/users/:username', getUserProfile);
+
```

Comment:
You are missing the 2 other endpoints:
1. Get repositories
2. Get users in a location.